### PR TITLE
[AXK1] Fix expected logits for CUDA A10G

### PR DESCRIPTION
**Pull request by ydshieh**
[CI](https://transformers-ci.lor-e.huggingface.cool/d/pytest-observability-by-pr/pytest-observability-branch?var-pr=47980)

## Summary

- `test_model_logits_batched` has been failing on A10G (CUDA compute capability 8.6) since the model was added in #46867
- The `("cuda", (8, 6))` expected values for both `EXPECTED_LOGITS_LEFT_PADDED` and `EXPECTED_LOGITS_UNPADDED` were incorrect (all 9 elements mismatched, greatest absolute diff ~0.47)
- Updated to values measured on A10G with PyTorch on the CI runner

## Test plan
- [x] `test_model_logits_batched` passes on A10G (verified on SSH CI runner)

### CODE REVIEW

**Comment by ydshieh** [comment]
run-slow: axk1
